content/platforms/release-notes/k8s-6-0-8-1-2020-10.md

```diff
@@ -0,0 +1,100 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.8-1 (October 2020)
+description: Release notes for version 6.0.8-1 (latest)
+weight: 74
+alwaysopen: false
+categories: ["Platforms"]
+---
+
+The Redis Enterprise K8s 6.0.8-1 release is a *major release* on top of [6.0.6-24](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.6-24) providing support for the latest [Redis Enterprise Software release 6.0.8-28](https://docs.redislabs.com/latest/rs/release-notes/rs-6-0-8-september-2020/) and includes several enhancements (including OpenShift 4.5 and Kubernetes 1.18 support) and bug fixes.
+
+## Overview
+
+This release of the operator provides:
+
+ * The latest release version of the operator
+ * New features
+ * New support K8s distributions and platforms
+ * Various bug fixes
+
+
+Follow the instructions in ["Upgrading a Redis Enterprise Cluster in Operator-based Architecture"]({{< relref "/platforms/kubernetes/upgrading-kubernetes-with-operator.md" >}}) to upgrade to this Kubernetes operator release.
+
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-28 or redislabs/redis:6.0.8-28.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-1
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-1 or redislabs/services-manager:6.0.8-1 (Red Hat registry)
+
+## New features
+
+ * Modules can be configuration in the database custom resource.
+ * Support was added for **OpenShift 4.5**
```

Comment:
want to add links to OpenShift 4.5 and K8s 1.18 on their respective Websites? perhaps to their release notes?

Reply:
I added the links